ibraheemosule/skoolizy
# za - Add PDF Viewer to Profile Page Using react-pdf-viewer

# Why this PR?
-This PR adds a PDF viewing and download feature on the profile page, allowing users to view PDF documents directly within the app and download them if needed. 

# What does this PR do?
-Users can view and download PDF documents directly within the profile page.

# How to test your work?
navigate to the my academic information section on the my profile page and click on the download and view icon to see the changes.

# Pages to see your changes?
/my-profile/academic-information
